models/model.js

```diff
@@ -1,14 +1,21 @@
 const db = require('../db/connection.js');
+const fs = require("fs/promises");
 
 function selectAllTopics (){
 
 return db.query(`SELECT * FROM topics`).then((topics) => {
     if (topics.rows.length === 0){
-        Promise.reject({status: 404, msg: 'Not Found'})
+        return Promise.reject({status: 404, msg: 'Not Found'})
     }
 return topics
 })
 
 }
 
-module.exports = selectAllTopics
\ No newline at end of file
+function selectAllEndpoints(){
```

Comment:
I like your naming convention where you've got `get` for the controller and `select` for the model, whilst keeping the nouns (e.g. `AllTopics`) consistent. Great work!